e3db/client.py

```diff
@@ -894,6 +893,42 @@ def __query(self, query):
         self.__response_check(response)
         return response.json()
 
+    def execute(self, query):
+        response = self.__execute(query)
+        results = response['results']
+        last_index = response['last_index']
+        search_id = response['search_id']
+
+        if results == None:
+            return QueryAdvResult(query, [], last_index, search_id)
+
+        records = []
+        for result in results:
+            result_meta = result['meta']
+            meta = Meta(result_meta)
+            result_data = result['record_data']
+            record = Record(meta=meta, data=result_data)
+            if query.include_data:
+                # need to decrypt all the results before returning.
```

Comment:
does this code exist elsewhere in this SDK(I would assume the existing search method does this) and if so can we re-use that method/refactor that code into a method and re-use it here?

Reply:
Yes It's actually a copy pasta from the previous search method, I'll pull it out and make it into its own method

Reply:
Don't forget to use the new method that you made here as well